### PR TITLE
Destination-Postgres: updated postgres.md documentation as per BETA template

### PR DESCRIPTION
**Pull request by etsybaev**
## What
Documentation didn't correspond to the template

## How
Updated documentation for connector

## 🚨 User Impact 🚨
No users impact

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/SUMMARY.md`
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Review by alexandr-shegeda — APPROVED** [review]
LGTM

File: docs/integrations/destinations/postgres.md [hunk 1]
```diff
@@ -49,50 +47,96 @@ GRANT CREATE, TEMPORARY ON DATABASE <database> TO airbyte_user;
...
+* Quoted identifiers can contain any character, except the character with code zero. \(To include a
+  double quote, write two double quotes.\) This allows constructing table or column names that would
+  otherwise not be possible, such as ones containing spaces or ampersands. The length limitation
+  still applies.
+* Quoting an identifier also makes it case-sensitive, whereas unquoted names are always folded to
+  lower case.
+* If you want to write portable applications you are advised to always quote a particular name or
+  never quote it.
+
+Therefore, Airbyte Postgres destination will create tables and schemas using the Unquoted
```
Comment (grishick) [inline docs/integrations/destinations/postgres.md:83]: ~~Therefore~~ Note, that

File: docs/integrations/destinations/postgres.md [hunk 2]
```diff
@@ -49,50 +47,96 @@ GRANT CREATE, TEMPORARY ON DATABASE <database> TO airbyte_user;
...
+| Incremental - Append Sync | Yes |  |
+| Incremental - Deduped History | Yes |  |
+| Namespaces | Yes |  |
 
-## Naming Conventions
+## Schema map
+
+#### Output Schema
+
+Each stream will be output into its own table in Postgres. Each table will contain 3 columns:
```
Comment (grishick) [inline docs/integrations/destinations/postgres.md:111]: Rephrase: Each stream will be mapped to a separate table in Postgres.

**Review by grishick — CHANGES_REQUESTED** [review]
I added some minor changes to wording in comments

File: docs/integrations/destinations/postgres.md [hunk 3]
```diff
@@ -1,37 +1,35 @@
...
 
-Airbyte Cloud only supports connecting to your Postgres instance with SSL or TLS encryption. TLS is used by default. Other than that, you can proceed with the open-source instructions below.
+* A Postgres server version 9.5 or above
 
-## Getting Started \(Airbyte Open-Source\)
+Airbyte Cloud only supports connecting to your Postgres instances with SSL or TLS encryption. TLS is
+used by default. Other than that, you can proceed with the open-source instructions below.
 
-#### Requirements
+You should now have all the requirements needed to configure Postgres as a destination in the UI.
```
[the remark this answers is not in the document]
Comment (etsybaev) [reply docs/integrations/destinations/postgres.md]: Updated, thanks!

File: docs/integrations/destinations/postgres.md [hunk 4]
```diff
@@ -49,50 +47,96 @@ GRANT CREATE, TEMPORARY ON DATABASE <database> TO airbyte_user;
...
-* **Password**
-* **Default Schema Name**
-* **Database**
-* **JDBC URL Params** (optional)
+* SQL identifiers and key words must begin with a letter \(a-z, but also letters with diacritical
+  marks and non-Latin letters\) or an underscore \(\_\).
+* Subsequent characters in an identifier or key word can be letters, underscores, digits \(0-9\), or
+  dollar signs \($\).
+
+  Note that dollar signs are not allowed in identifiers according to the letter of the SQL standard,
```
[the remark this answers is not in the document]
Comment (etsybaev) [reply docs/integrations/destinations/postgres.md]: Updated, thanks!

File: docs/integrations/destinations/postgres.md [hunk 5]
```diff
@@ -49,50 +47,96 @@ GRANT CREATE, TEMPORARY ON DATABASE <database> TO airbyte_user;
...
+* The system uses no more than NAMEDATALEN-1 bytes of an identifier; longer names can be written in
+  commands, but they will be truncated. By default, NAMEDATALEN is 64 so the maximum identifier
+  length is 63 bytes
+* Quoted identifiers can contain any character, except the character with code zero. \(To include a
+  double quote, write two double quotes.\) This allows constructing table or column names that would
+  otherwise not be possible, such as ones containing spaces or ampersands. The length limitation
+  still applies.
+* Quoting an identifier also makes it case-sensitive, whereas unquoted names are always folded to
+  lower case.
+* If you want to write portable applications you are advised to always quote a particular name or
```
[the remark this answers is not in the document]
Comment (etsybaev) [reply docs/integrations/destinations/postgres.md]: Updated, thanks!

File: docs/integrations/destinations/postgres.md (hunk 1 above)
Comment (etsybaev) [reply docs/integrations/destinations/postgres.md]: Updated, thanks!

**Review by grishick — APPROVED** [review]

**Pull request merged**